src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java

```diff
@@ -75,46 +84,50 @@ public CommandManagerWindow(DriverCommandManager commandManager) {
 		c.fill = GridBagConstraints.BOTH;
 		c.weightx = 10;
 		c.gridx = 1;
-		c.weighty = 20;
+		c.gridy = 0;
+		c.weighty = 1;
+		c.gridheight = 5;
+		c.gridwidth = 1;
 		content.add(previewPanel, c);
-
+		preview = new CommandManagerPreview(previewPanel, new BasicLine());
```

Comment:
dependencies from CommandManagerPreview burden CommandManagerWindow. You didn't manage to inverse dependencies.